applications/fastapi-bootcamp/templates/ingress.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: gafaelfawr.lsst.io/v1alpha1
+kind: GafaelfawrIngress
+metadata:
+  name: "fastapi-bootcamp"
+  labels:
+    {{- include "fastapi-bootcamp.labels" . | nindent 4 }}
+config:
+  baseUrl: {{ .Values.global.baseUrl | quote }}
+  scopes:
+    all:
+      - "read:image"
+  loginRedirect: true
+template:
+  metadata:
+    name: "fastapi-bootcamp"
+    {{- with .Values.ingress.annotations }}
+    annotations:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+  spec:
+    rules:
+      - host: {{ required "global.host must be set" .Values.global.host | quote }}
+        http:
+          paths:
+            - path: "/fastapi-bootcamp"
```

Use the configuration instead of separate hard coding:

```suggestion
            - path: {{ .Values.config.pathPrefix | quote }}
```